ministryofjustice/laa-review-criminal-legal-aid
# CRIMRE-273 - Changed wording from 'View all open applications' link to 'Check all open applications' link on the 'Your list' page

## Description of change
Changed wording from 'View all open applications' link to 'Check all open applications' link on the 'Your list' page

## Link to relevant ticket
https://dsdmoj.atlassian.net/browse/CRIMRE-273

## Notes for reviewer

## Screenshots of changes (if applicable)

### Before changes:
<img width="482" alt="image" src="https://user-images.githubusercontent.com/106679087/232471593-65ed7fdb-0e48-4916-b795-005d768880f2.png">

### After changes:
![image](https://user-images.githubusercontent.com/106679087/232471400-b1be6330-735e-4037-993f-36cab3a0701f.png)

## How to manually test the feature
- User logs in the portal
- User will see the 'Your list' page
- User will see the 'Check all open applications' link as shown above in the After changes screenshot
